src/openforms/authentication/api/serializers.py

```diff
@@ -30,6 +30,22 @@ class AuthPluginSerializer(PluginBaseSerializer):
     )
 
 
+class AttributeGroupSerializer(serializers.Serializer):
```

Comment:
this does not belong in the `openforms.authentication` namespace, but in `openforms.authentication.contrib.yivi`. Currently this is violating the self-containment idea - we organize code in django apps/packages to maintain a predictable structure. See https://open-forms.readthedocs.io/en/stable/developers/backend/index.html#general-principles: "Keep Django apps contained".

The pattern is similar to plugin-specific endpoints in registrations: https://github.com/open-formulieren/open-forms/blob/b04a5c3d8e7178e318700f12b094ae6a61fa7b6d/src/openforms/registrations/api/urls.py#L14

Additionally, for the actual code - why can't you use `ModelSerializer`?